server/resources/migrations/26_cli_auth.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE instant_cli_auth (
+  id uuid PRIMARY KEY,
+  secret uuid NOT NULL,
+  used boolean NOT NULL DEFAULT false,
+  created_at timestamp NOT NULL DEFAULT now(),
+  user_id uuid REFERENCES instant_users(id) ON DELETE CASCADE
```

Comment:
Can you add an index on `user_id`? That way when a user is deleted, postgres doesn't have to do a full table scan on the `instant_cli_auth` table.